graph/typeDefs.graphql

```diff
@@ -140,6 +140,9 @@ input CommentsQuery {
 
   # Sort the results by created_at.
   sort: SORT_ORDER = REVERSE_CHRONOLOGICAL
+
+  # Exclude comments ignored by this user ID
+  notIgnoredBy: String
```

Comment:
This should only ever be the current user, why not inherit this from the context instead? It always contains the current logged in user (unless the user is not logged in)